### ISSUE TITLE
(next branch) Gulp install task uses default output paths for first (post-install) build

### ISSUE THREAD

**Issue by kevindice** [Semantic-Org/Semantic-UI#2398]
Reproduction steps:
(Starting with a fresh copy of the next branch)

1. npm install
2. Once the gulp install task starts, select the third option (custom)
3. Specify custom output paths
4. Select yes when prompted for building the project
5. Observe output paths used

All subsequent builds will load the custom paths from semantic.json, but the optional post-install build does not.

**Comment by jlukic** [comment]
Thanks, I'll take a look at this tomorrow

**Comment by jlukic** [comment]
I really should stop saying tomorrow. Either way this is still in queue.

**Comment by jlukic** [comment]
I don't believe it should ask you to build files now when doing `npm install`.

It should only do this when running `gulp install` after forking the repo (different install path).
